### PR TITLE
Added CardDTO and ProcessPayment endpoint

### PR DESCRIPTION
**Pull request by epadams**
- Implemented CardDTO
- Implemented ProcessPayment endpoint with error checking (could be reduced/cleaned, change to a switch statement?)

### CODE REVIEW

File: ShopAPI/ShopAPI/DTOs/CardDTO.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+namespace ShopAPI.DTOs
+{
+    public class CardDTO
+    {
+        public int CartId { get; set; }
+        public long CardNumber { get; set; }
+        public string Exp { get; set; }
+        public string CardHolderName { get; set; }
+        public int Cvv { get; set; }
+    }
```
Comment (evan-scales) [inline ShopAPI/ShopAPI/DTOs/CardDTO.cs:10]: Lets also rename this to CheckoutDTO or something similar.

Comment (evan-scales) [reply ShopAPI/ShopAPI/DTOs/CardDTO.cs:10]: Silly me, I thought it said CartDTO. CardDTO is fine

**Comment by epadams** [comment]
I don't see a good way to do this with a switch statement. It seems like switch statements are mainly for decision making based on properties or static operations like arithmetic. I'll try to do some more error checking with the expiration date.

**Comment by evan-scales** [comment]
Dont worry too much about using a switch statement, as long as you find a way to make sure an expiration date is good

**Review by evan-scales — APPROVED** [review]
looks good

**Pull request merged**